avtozip/store/api/resources.py

```diff
@@ -0,0 +1,16 @@
+from tastypie.resources import ModelResource
+from store.models import Product
+from django.http import Http404
+
+
+class ProductResource(ModelResource):
+    class Meta:
+        queryset = Product.objects.all()
+        allowed_methods = ['post', 'put', 'get', 'patch', 'delete']
+        filtering = {
+            'name': ('exact', 'icontains',),
```

Comment:
I've checked this before. You'll get exception when `exact` is absent and you request a data with `name=<something>` to API